rotkehlchen/history/manager.py

```diff
@@ -233,14 +236,13 @@ def new_step_cb(state_name: str) -> None:
         # Include all base history entries
         history_events_db = DBHistoryEvents(self.db)
         with self.db.conn.read_ctx() as cursor:
-            base_entries = history_events_db.get_history_events(
+            base_entries = history_events_db.get_history_events_internal(
```

Comment:
can you keep the comment about applying the limit when the events get processed?